lib/reverse_words.rb

```diff
@@ -1,4 +1,21 @@
 # A method to reverse each word in a sentence, in place.
 def reverse_words(my_words)
-  raise NotImplementedError
+  starting = 0
+  ending = 0
+  # put in a case for nil no word
+  while my_words != nil && ending <= my_words.length
```

Comment:
I believe you'd want to end the loop when ending becomes equal to my_words.length. So the condition should be `while my_words != nil && ending < my_words.length`